app/config/MicroserviceAppConfig.scala

```diff
@@ -34,8 +34,6 @@ class MicroserviceAppConfig @Inject()(servicesConfig: ServicesConfig) {
     )
   }
 
-  val isIfEnabled: Boolean = servicesConfig.getBoolean("microservice.services.if.enabled")
```

Comment:
There are a few references to this config value in FinancialDetailPaymentsControllerISpec and FinancialDetailsControllerISpec, can you remove them as well?